pkg/collector/corechecks/snmp/internal/common/utils.go

```diff
@@ -28,3 +28,18 @@ func CreateStringBatches(elements []string, size int) ([][]string, error) {
 
 	return batches, nil
 }
+
+// AreTagsEqual compares two list of tags supposing that tags are in the same order
+func AreTagsEqual(tagsA, tagsB []string) bool {
```

Comment:
Could you use https://pkg.go.dev/reflect#DeepEqual instead of adding this?

Reply:
yes ! https://github.com/DataDog/datadog-agent/pull/25534/commits/4174011a910ae2b72866da6e965118b820dd11b3